src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -14,15 +14,16 @@ import { useVenueId } from "hooks/useVenueId";
 
 import "./UserAvatar.scss";
 
+export type UserAvatarSize = "legacy" | "small" | "medium" | "large" | "full";
+
 export interface UserAvatarProps {
   user?: WithId<User>;
   containerClassName?: string;
   imageClassName?: string;
   showNametag?: UsernameVisibility;
   showStatus?: boolean;
   onClick?: () => void;
-  large?: boolean;
-  medium?: boolean;
+  size?: UserAvatarSize;
 }
```

Comment:
Setting a `size` prop with limited set of values helps not bloat the `UserAvatarProps` with many mutually exclusive booleans, keeps the possibility of being expanded, but also limits the variations to a handful for consistency and maintainability.

Reply:
@goran-peoski-work I don't think we should add `legacy`. Let's just leave the default as it used to be, but add it to our map, which should equal to medium

Reply:
'legacy' is the word for default. the default is already left as it used to be, I just didn't want to use that label in case there's new default

Reply:
@goran-peoski-work the `default` should be not passed value, so there should be no string `default`/`legacy`. Not passing a size option means it's the default. What do you mean by `new default`?

Reply:
`legacy` does sound strange for a prop called `size`. I might be wrong here, but looks like there was no default value before, why do we need it set to `legacy` now?

Reply:
removed legacy value @ 76220e8c2aab97e736091f15773d4ae204e4ddd7